var/spack/repos/builtin/packages/py-lxml/package.py

```diff
@@ -44,3 +45,4 @@ class PyLxml(PythonPackage):
     depends_on("py-html5lib", when="+html5", type=("build", "run"))
     depends_on("py-beautifulsoup4", when="+htmlsoup", type=("build", "run"))
     depends_on("py-cssselect@0.7:", when="+cssselect", type=("build", "run"))
+    depends_on("py-cython@3.0.10:", type="build", when="@5.2.2:")
```

Comment:
This is likely starting when `@5:` right?

Reply:
No. There are minor version bumps with some releases and not others. I verified 5.2.2. If/when someone adds previous versions they will have to check. Release notes are a bit patchy so I could not readily find e.g. when v3 started to be needed. (I have no connection to the lxml project.)

Reply:
I think the following is sufficient for versions 5, based on git blames of `pyproject.toml` and `requirements.txt`.
```suggestion
    depends_on("py-cython@3.0.10:", type="build", when="@5.2:")
    depends_on("py-cython@3.0.9:", type="build", when="@5.1.1:")
    depends_on("py-cython@3.0.8:", type="build", when="@5:")
    depends_on("py-cython@0.29.7:", type="build")
```
For older versions, `py-cython` may also have been required per `requirements.txt`, but never added. Version 0.29.7 is sufficiently old that I think we can safely require it for versions older than 4.4 as well (when that version requirement was last bumped up).